overrides/partials/footer.html

```diff
@@ -52,8 +52,10 @@
         </a>
       </div>
       <div class="md-footer-copyright" style="display: flex; align-items:center;">
-        <span style="font-size:14px; margin-right: 8px;">Founded by</span>
-        <img style="width:120px" alt="microsoft-logo" src="https://microsoft.github.io/electionguard/assets/images/microsoft-logo.png"/>
+        <ul>
```

Comment:
If this is an unordered list it needs to remove the styling. 
![Screen Shot 2021-03-01 at 2 06 56 PM](https://user-images.githubusercontent.com/10125297/109545940-84b46f80-7a97-11eb-8427-9da6ea50f8b7.png)

`list-style: none`